mozilla-release/mobile/android/tests/cliqz/java/com/cliqz/mobile/test/pages/FreshTabPage.java

```diff
@@ -0,0 +1,196 @@
+package com.cliqz.mobile.test.pages;
+
+import android.support.test.uiautomator.By;
+import android.support.test.uiautomator.BySelector;
+import android.support.test.uiautomator.UiObject2;
+
+import com.cliqz.mobile.test.CliqzRunner;
+import com.cliqz.mobile.test.driver.NativeDriver;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+@RunWith(CliqzRunner.class)
+public class FreshTabPage extends BasePage {
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
```

Comment:
Variable declarations before any methods

Reply:
reordered the class.